bait-files/src/main/webapp/WEB-INF/profile.jsp

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
This file is not needed for this PR right?

Reply:
Removed